test/unittests/state_transition.cpp

```diff
@@ -42,20 +42,21 @@ void state_transition::TearDown()
             ASSERT_NE(acc, nullptr) << "account " << addr << " should exist";
             if (expected_acc.nonce.has_value())
             {
-                EXPECT_EQ(acc->nonce, *expected_acc.nonce);
+                EXPECT_EQ(acc->nonce, *expected_acc.nonce) << "account " << addr;
             }
             if (expected_acc.balance.has_value())
             {
                 EXPECT_EQ(acc->balance, *expected_acc.balance)
-                    << to_string(acc->balance) << " vs " << to_string(*expected_acc.balance);
+                    << to_string(acc->balance) << " vs " << to_string(*expected_acc.balance)
+                    << " account " << addr;
```

Comment:
Is it possible to ass unit test here? I think I was adding in a different PR

Reply:
I doubt so, as this is a fixture code... other lines here are also not tested, they are just "covered" because they are being run during (succeeding) tests in unit test sute.